### DISCUSSION TITLE
Ability to bulk edit arbitrary EXIF Information

### DISCUSSION THREAD

**Discussion by DamageIncorporated** [ente-io/ente#2360] [Enhancements]
I'm following up to another discussion I opened here: https://github.com/ente-io/ente/discussions/2359.

After adding a bunch of photos, I would like to be able to add tags, say from a vacation, festival, etc.

Now I would like to be able to add a single description, tags, maybe geo information (if it's not already present) to all selected photos.

That would overwrite (old) information that's already present in a specific field, which is sometimes exactly what I want, sometimes an accident, so a short warning would be useful ;-)

In the old times, when using mp3 taggers, there was a checkmark next to every data field, that you would check to show that you want to overwrite this specific value for every file. 
If you leave it unchecked, that field won't be touched when updating the stack of files...

So far the idea :-)

**Comment by kohloderso** [comment]
I'd also like this feature to organize the photos better.
Right now I'm, not quite sure how best to organize my storage. On my PC I had one large folder per year, containing many small folders for events that happened in that year. Since Ente doesn't support nested folders, I now have just one large album for each year. If I could simply select all pictures belonging to an event and bulk edit some description, that would help a lot!
